source/wp-content/themes/wporg-parent-2021/patterns/heading-with-arrow-and-list.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Title: Heading with arrow and list
+ * Slug: wporg-parent-2021/heading-with-arrow-and-list
+ * Categories: header
+ */
```

Comment:
I'm not sure if we need 3 variations of this pattern, the original issue was just to create the heading & arrow. I don't feel strongly about that though, so if you think all 3 are useful we can keep them all in.

My comments about the block structure apply to all 3.

Reply:
I'm curious who would eventually use these patterns?
Would the users be ourselves or other developers only? If it would be used by some non-tech guys, I'm thinking that providing the 3 variations would still save much time for them.

Reply:
Cool, I'll remove them.